src/markdown-pages/collect-data/opentelemetry-manual/view.mdx

```diff
@@ -0,0 +1,195 @@
+---
+duration: 5
+title: 'View your OpenTelemetry data in New Relic'
+template: 'GuideTemplate'
+description: 'View your OpenTelemetry data in New Relic'
+procIdx: 3
+---
+
+<Callout variant="course" title="lab">
+
+This procedure is part of a lab that teaches you how to instrument your application with OpenTelemetry.
+
+Each procedure in the lab builds upon the last, so make sure you've completed the last procedure, [_Instrument your application with OpenTelemetry_](/collect-data/opentelemetry-manual/instrument), before starting this one.
+
+</Callout>
+
+You've instrumented your database application with OpenTelemetry and you're sending trace data to New Relic.
+
+Here, you move to New Relic to see the kinds of detailed telemetry data that you generated with just a few lines of OpenTelemetry code.
+
+## View your data
+
+<Steps>
+
+<Step>
+
+Log into [New Relic](https://one.newrelic.com).
+
+</Step>
+
+<Step>
+
+In the entity explorer, click the **speedeedeebee** OpenTelemetry service:
```

Comment:
The image here shows the entities last viewed. It's highly unlikely that somebody who just spun up the application will see the service here. Instead of callout, can we move the navigation part here already? OpenTelemetry service > speedeedeebee